src/comonents/Header/Header.scss

```diff
@@ -0,0 +1,200 @@
+@import '../../prepare-styles/mixins.scss';
+
+@font-face {
+  font-family: Mont-bold;
+  src: url(../../../public/fonts/Mont-Bold.otf);
+}

```

Comment:
probably better remove this now or later, as we already have separate file to import fonts